extras/AFC_led.py

```diff
@@ -128,21 +128,31 @@ def lookahead_bgfunc(print_time):
             else:
                 set_color_fn = self.led_helper.set_color
                 check_transmit_fn = self.led_helper.check_transmit
-            set_color_fn(index, colors)
+
+            if "-" in index:
```

Comment:
This should be the right change to make and revert changes in `turn_off_leds` and `turn_on_leds` functions
```suggestion
            if isinstance(index, str) and "-" in index:
```